packages/extension-ui/src/Popup/AuthManagement/WebsiteEntry.tsx

```diff
@@ -124,13 +124,15 @@ export default styled(WebsiteEntry)(
     height: 20px;
   }
 
-  .chevron path{
-    fill: ${theme.iconNeutralColor};
+  .chevron {
+    width: 16px;
+    height: 16px;
+    background: ${theme.iconNeutralColor};
   }
 
   &:hover { 
-    .chevron path{
-      fill: ${theme.headerIconBackgroundHover};
+    .chevron {
+      background: ${theme.headerIconBackgroundHover};
     }
   }
```

Comment:
It's not anything very important, but sometimes if the selector is as simple nesting is not really required
```css
&:hover .chevron {
  background: ${theme.headerIconBackgroundHover};
}
```